Sources/KIF/Additions/UIAccessibilityCustomAction+KIFAdditions.h

```diff
@@ -0,0 +1,15 @@
+//
+//  UIAccessibilityCustomAction+KIFAdditions.h
+//  KIF
+//
+//  Created by Alex Odawa on 09/07/2024.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface UIAccessibilityCustomAction (KIFAdditions)
+
+- (BOOL)activate;
```

Comment:
Use a prefix to avoid collisions, e.g. `KIF_activate`

Reply:
oh man, i honestly do love objective c